### PR TITLE
Update NetworkManager

### PR DESCRIPTION
**Pull request by mahantesh-ais**

### CODE REVIEW

**Comment by mahantesh-ais** [comment]
To be merged along with:
https://github.com/OpenXT/network/pull/14

File: recipes-security/refpolicy/refpolicy-mcs-2.%/patches/add-missing-dbusd-permissions.patch [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
++allow system_dbusd_t self:process { getattr getsched signal_perms setpgid getcap setcap setrlimit setsched };
+ allow system_dbusd_t self:fifo_file rw_fifo_file_perms;
+ allow system_dbusd_t self:dbus { send_msg acquire_svc };
+ allow system_dbusd_t self:unix_stream_socket { accept connectto listen };
+@@ -88,6 +88,8 @@ can_exec(system_dbusd_t, dbusd_exec_t)
+ 
+ kernel_read_system_state(system_dbusd_t)
+ kernel_read_kernel_sysctls(system_dbusd_t)
++kernel_read_vm_overcommit_sysctl(system_dbusd_t)
++kernel_search_vm_sysctl(system_dbusd_t)
```
Comment (jandryuk) [inline recipes-security/refpolicy/refpolicy-mcs-2.%/patches/add-missing-dbusd-permissions.patch]: `kernel_read_vm_overcommit_sysctl` contains `kernel_search_vm_sysctl, so you can drop `kernel_search_vm_sysctl`

I'm a little surprised that updating Network Manager requires a change to dbus's permissions.

**Comment by jean-edouard** [comment]
Built here: http://openxt-builder.ainfosec.com:8010/builders/openxt64/builds/6584

File: recipes-security/refpolicy/refpolicy-mcs-2.%/patches/add-missing-dbusd-permissions.patch (hunk 1 above)
Comment (mahantesh-ais) [reply recipes-security/refpolicy/refpolicy-mcs-2.%/patches/add-missing-dbusd-permissions.patch]: Ack, will drop `kernel_search_vm_sysctl`.

Yes, I looked in the code to see if Network Manager triggers any new dbus requests or something. I couldn't find any. But, without these permissions some of the functionalities won't work, so these are required and being used by Network Manager.

**Comment by jean-edouard** [comment]
LGTM, merging soon.

**Pull request merged**